src/Centreon/Domain/Monitoring/Resource.php

```diff
@@ -491,9 +496,9 @@ public function setCommandLine(?string $commandLine): self
     }
 
     /**
-     * @return string
+     * @return string|null
      */
-    public function getMonitoringServerName(): string
+    public function getMonitoringServerName(): ?string
```

Comment:
Is it possible that the monitoring server does not have a name?

Reply:
Nope you are right. The monitoring server name can't be NULL.